geppetto.js/geppetto-client/src/common/layout/WidgetFactory.tsx

```diff
@@ -56,7 +56,9 @@ class WidgetFactory {
     }
     return Component ?
       <Component
+        id={widgetConfig.id}
         key={widgetConfig.id}
+        model={widgetConfig.model}
```

Comment:
@ddelpiano did you try to move the model inside config?

Reply:
https://github.com/MetaCell/geppetto-meta/pull/255/commits/1c0318c7410c2d8faa6dc3d53bb3ea6c76629947